presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -100,7 +100,24 @@ public static QueryRunner createQueryRunner(boolean addStorageFormatToPath, bool
                 cacheMaxSize,
                 DEFAULT_STORAGE_FORMAT,
                 addStorageFormatToPath,
-                isCoordinatorSidecarEnabled);
+                isCoordinatorSidecarEnabled,
+                false);
+    }
+
+    public static QueryRunner createQueryRunner(boolean enableRuntimeMetricsCollection)
```

Comment:
would be better to add this parameter to the existing `createQueryRunner` function:
```
public static QueryRunner createQueryRunner(boolean addStorageFormatToPath, boolean isCoordinatorSidecarEnabled)
```